### PR TITLE
Add tests for sycl::ext::oneapi::property_value

### PR DESCRIPTION
**Pull request by kopylovanat**
sycl::ext::oneapi::property_value is part of extension [SYCL_EXT_ONEAPI_PROPERTY_LIST](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/PropertyList/SYCL_EXT_ONEAPI_property_list.asciidoc)
Extension [SYCL_EXT_ONEAPI_DEVICE_GLOBAL](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/DeviceGlobal/SYCL_INTEL_device_global.asciidoc) is also used to check  how compile-time-constant properties work.

### CODE REVIEW

**Review by keryell — COMMENTED** [review]
So what is the current decision about where we put tests from an extension?
In a `oneapi` subdirectory somewhere mimicking the normal test directory so we can look quickly at all the tests from an extension easily?

**Comment by bader** [comment]
> So what is the current decision about where we put tests from an extension? 

This question was raised in https://github.com/KhronosGroup/SYCL-CTS/pull/200 and I see that @vasilytric created a separate directory in that PR. I suggest we agree on the location there and then apply it to other PRs. @keryell, does it sound okay to you?

**Comment by kopylovanat** [comment]
> So what is the current decision about where we put tests from an extension?
> In a `oneapi` subdirectory somewhere mimicking the normal test directory so we can look quickly at all the tests from an extension easily?

Moved tests to subfolder `extension` similarly to how it done in  https://github.com/KhronosGroup/SYCL-CTS/pull/200

File: docker/computecpp/configure.sh [hunk 1]
```diff
@@ -7,4 +7,5 @@ cmake . -G Ninja -B build \
     -DCMAKE_PREFIX_PATH=/sycl \
     -DCMAKE_BUILD_TYPE=Release \
     -DSYCL_CTS_ENABLE_FULL_CONFORMANCE=0 \
+    -DSYCL_ENABLE_EXT_ONEAPI_PROPERTY_LIST_TESTS=0 \
```
Comment (ProGTX) [inline docker/computecpp/configure.sh:10]: All extension tests should be off by default, so we don't have to specify the variables for the implementations that don't support them.

File: docker/dpcpp/configure.sh [hunk 2]
```diff
@@ -8,4 +8,5 @@ cmake . -G Ninja -B build \
     -DCMAKE_CXX_COMPILER=/sycl/bin/clang++ \
     -DCMAKE_BUILD_TYPE=Release \
     -DSYCL_CTS_ENABLE_FULL_CONFORMANCE=0 \
+    -DSYCL_ENABLE_EXT_ONEAPI_PROPERTY_LIST_TESTS=0 \
```
Comment (ProGTX) [inline docker/dpcpp/configure.sh:11]: Is this intentional? Presumably DPC++ supports this extension?

**Review by ProGTX — APPROVED** [review]

File: docker/computecpp/configure.sh (hunk 1 above)
Comment (kopylovanat) [reply docker/computecpp/configure.sh]: Removed.

File: docker/dpcpp/configure.sh (hunk 2 above)
Comment (kopylovanat) [reply docker/dpcpp/configure.sh]: These tests are off by default for all compilers. so removed this too.

**Review by keryell — APPROVED** [review]
Good!

**Comment by fraggamuffin** [comment]
LGTM

**Comment by keryell** [comment]
There are some merge conflicts to fix.

**Pull request merged**